Benchmarks/Benchmarks/Library.json

```diff
@@ -791,15 +797,21 @@
           "title": "insert",
           "tasks": [
             "Heap<Int> insert",
-            "Heap<Int> insert(contentsOf:)"
+            "Heap<Int> insert(contentsOf:)",
+            "PriorityQueue<Int, Int> insert",
+            "PriorityQueue<Task, Int> insert"
```

Comment:
@lorentey Should we pull the priority queue stuff out into a separate library JSON? (Or add a separate priority queue library JSON in addition to adding these benchmarks here?)